tokio-async-await/src/async_await/io/mod.rs

```diff
@@ -0,0 +1,192 @@
+//! Use I/O with `async` / `await`.
+
+mod flush;
+mod read;
+mod read_exact;
+mod write;
+mod write_all;
+
+pub use self::flush::Flush;
+pub use self::read::Read;
+pub use self::read_exact::ReadExact;
+pub use self::write::Write;
+pub use self::write_all::WriteAll;
+
+use tokio_io::{AsyncRead, AsyncWrite};
+
+/// An extension trait which adds utility methods to `AsyncRead` types.
+pub trait AsyncReadExt: AsyncRead {
+    /// Tries to read some bytes directly into the given `buf` in asynchronous
```

Comment:
This should probably read "Tries to read some bytes directly into the given `buf` in _an_ asynchronous manner".